airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/TemporalClient.java

```diff
@@ -378,11 +380,32 @@ public TemporalResponse<StandardSyncOutput> submitSync(final long jobId, final i
         .withDockerImage(config.getSourceDockerImage())
         .withProtocolVersion(config.getSourceProtocolVersion());
 
-    final IntegrationLauncherConfig destinationLauncherConfig = new IntegrationLauncherConfig()
-        .withJobId(String.valueOf(jobId))
-        .withAttemptId((long) attempt)
-        .withDockerImage(config.getDestinationDockerImage())
-        .withProtocolVersion(config.getDestinationProtocolVersion());
+    IntegrationLauncherConfig destinationLauncherConfig;
+    try {
+      LocalDefinitionsProvider provider = new LocalDefinitionsProvider(LocalDefinitionsProvider.DEFAULT_SEED_DEFINITION_RESOURCE_CLASS);
+      List<StandardDestinationDefinition> destinationDefinitionList = provider.getDestinationDefinitions();
```

Comment:
We shouldn't be reading from a definitions provider or direct yamls here- they're mainly used to seed/update the connector definitions in the database (actor_definitions table) and then any other operations within the platform should use the data in the db. For example, cloud doesn't use local definitions and instead loads remote definitions from a service.

I think by this point we should have the normalization tag / repo / anything else in that database table, specially since I see a migration adding those columns to the db... is this true?

Reply:
Yes you're right. This is a very good point. Thank you for it. I changed this piece of code and now I get the data I need from the database. Also, I removed this logic from **TemporalClient.class** - as far as I understand, that part working on the synchronization process for which we may not retrieve this information.